linchpin/provision/roles/azure/tasks/provision_azure_vm.yml

```diff
@@ -70,7 +70,7 @@
     tenant: "{{ auth_var['tenant'] | default(omit) }}"
     secret: "{{ auth_var['secret'] | default(omit) }}"
     subscription_id: "{{ auth_var['subscription_id'] | default(omit) }}"
-    admin_username: "{{ res_def['vm_username'] | default('{{linchpinUsername}}') }}"
+    admin_username: "{{ res_def['vm_username'] | default(linchpinUsername) }}"
```

Comment:
@junqizhang Where are linchpinUsername and linchpinPassword coming from ???

Reply:
Shouldnt they be in single quotes ??

Reply:
They r coming from the defaults.yml